sdk/program/src/lamports.rs

```diff
@@ -1,23 +1,2 @@
 //! Defines the [`LamportsError`] type.
-
-use {crate::instruction::InstructionError, thiserror::Error};
-
-#[derive(Debug, Error)]
-pub enum LamportsError {
-    /// arithmetic underflowed
-    #[error("Arithmetic underflowed")]
-    ArithmeticUnderflow,
-
-    /// arithmetic overflowed
-    #[error("Arithmetic overflowed")]
-    ArithmeticOverflow,
-}
-
-impl From<LamportsError> for InstructionError {
-    fn from(error: LamportsError) -> Self {
-        match error {
-            LamportsError::ArithmeticOverflow => InstructionError::ArithmeticOverflow,
-            LamportsError::ArithmeticUnderflow => InstructionError::ArithmeticOverflow,
-        }
-    }
-}
+pub use solana_instruction::error::LamportsError;
```

Comment:
Should we have a `deprecated` tag here?

Reply:
this one was a coinflip for me. I'll add a deprecated tag